### PR TITLE
Bug fix for network_test

### PR DESCRIPTION
**Pull request by vaishnavibhat**
network_test.py is failing with
ERROR: 'NetworkTest' object has no attribute 'ping_check'

Signed-off-by: Vaishnavi Bhat <vaishnavi@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by vaishnavibhat** [comment]
> @vaishnavibhat
> if not HostInfo.ping_check(self, self.iface, self.peer, "2"):
> in place of that this need to written.
> while i am sending i think i forgetted to edit that.

There is already ping check done in the earlier part of the code in setUp(self).
      if not HostInfo.ping_check(self, self.iface, self.peer, "2"):
            self.cancel("No connection to peer")

**Comment by narasimhan-v** [comment]
@vaishnavibhat job.log please.

**Comment by manvanthar** [comment]
Agree there are two locations where the ping is checked in setup.  One of it is irrelavent and this patch should fix it.

**Comment by vaishnavibhat** [comment]
[root@ltcfleet2-lp3 net]# avocado run network_test.py -m network_test.py.data/network_test.yaml
JOB ID : cdf5864938aeedba68cb68ac21c5a362b0c12c86
JOB LOG : /root/avocado-fvt-wrapper/results/job-2019-11-20T19.47-cdf5864/job.log
(01/81) network_test.py:NetworkTest.test_gro;1500-d20e: PASS (11.90 s)
(02/81) network_test.py:NetworkTest.test_gro;2000-3295: PASS (12.49 s)
(03/81) network_test.py:NetworkTest.test_gro;3000-8d15: PASS (12.43 s)
(04/81) network_test.py:NetworkTest.test_gro;4000-a8bf: PASS (12.37 s)
(05/81) network_test.py:NetworkTest.test_gro;5000-fe4f: PASS (12.41 s)
(06/81) network_test.py:NetworkTest.test_gro;6000-da64: PASS (12.62 s)
(07/81) network_test.py:NetworkTest.test_gro;7000-354d: PASS (12.35 s)
(08/81) network_test.py:NetworkTest.test_gro;8000-257a: PASS (12.47 s)
(09/81) network_test.py:NetworkTest.test_gro;9000-74de: PASS (12.33 s)
(10/81) network_test.py:NetworkTest.test_lro;1500-d20e: FAIL: lro on failed (7.75 s)
(11/81) network_test.py:NetworkTest.test_lro;2000-3295: FAIL: lro on failed (8.42 s)
(12/81) network_test.py:NetworkTest.test_lro;3000-8d15: FAIL: lro on failed (8.35 s)
(13/81) network_test.py:NetworkTest.test_lro;4000-a8bf: FAIL: lro on failed (8.38 s)
(14/81) network_test.py:NetworkTest.test_lro;5000-fe4f: FAIL: lro on failed (8.39 s)
(15/81) network_test.py:NetworkTest.test_lro;6000-da64: FAIL: lro on failed (8.45 s)
(16/81) network_test.py:NetworkTest.test_lro;7000-354d: FAIL: lro on failed (8.43 s)
(17/81) network_test.py:NetworkTest.test_lro;8000-257a: FAIL: lro on failed (8.43 s)
(18/81) network_test.py:NetworkTest.test_lro;9000-74de: FAIL: lro on failed (8.52 s)
(19/81) network_test.py:NetworkTest.test_ping;1500-d20e: PASS (7.58 s)
(20/81) network_test.py:NetworkTest.test_ping;2000-3295: PASS (8.36 s)
(21/81) network_test.py:NetworkTest.test_ping;3000-8d15: PASS (8.17 s)
(22/81) network_test.py:NetworkTest.test_ping;4000-a8bf: PASS (8.29 s)
(23/81) network_test.py:NetworkTest.test_ping;5000-fe4f: PASS (8.27 s)
(24/81) network_test.py:NetworkTest.test_ping;6000-da64: PASS (8.28 s)
(25/81) network_test.py:NetworkTest.test_ping;7000-354d: PASS (8.21 s)
(26/81) network_test.py:NetworkTest.test_ping;8000-257a: PASS (8.18 s)
(27/81) network_test.py:NetworkTest.test_ping;9000-74de: PASS (8.20 s)
(28/81) network_test.py:NetworkTest.test_floodping;1500-d20e: PASS (3.45 s)
(29/81) network_test.py:NetworkTest.test_floodping;2000-3295: PASS (4.12 s)
(30/81) network_test.py:NetworkTest.test_floodping;3000-8d15: PASS (4.11 s)
(31/81) network_test.py:NetworkTest.test_floodping;4000-a8bf: PASS (4.18 s)
(32/81) network_test.py:NetworkTest.test_floodping;5000-fe4f: PASS (4.10 s)
(33/81) network_test.py:NetworkTest.test_floodping;6000-da64: PASS (4.16 s)
(34/81) network_test.py:NetworkTest.test_floodping;7000-354d: PASS (4.07 s)
(35/81) network_test.py:NetworkTest.test_floodping;8000-257a: PASS (4.00 s)
(36/81) network_test.py:NetworkTest.test_floodping;9000-74de: PASS (4.12 s)
(37/81) network_test.py:NetworkTest.test_ssh;1500-d20e: PASS (3.67 s)
(38/81) network_test.py:NetworkTest.test_ssh;2000-3295: PASS (4.35 s)
(39/81) network_test.py:NetworkTest.test_ssh;3000-8d15: PASS (4.40 s)
(40/81) network_test.py:NetworkTest.test_ssh;4000-a8bf: PASS (4.31 s)
(41/81) network_test.py:NetworkTest.test_ssh;5000-fe4f: PASS (4.37 s)
(42/81) network_test.py:NetworkTest.test_ssh;6000-da64: PASS (4.35 s)
(43/81) network_test.py:NetworkTest.test_ssh;7000-354d: PASS (4.28 s)
(44/81) network_test.py:NetworkTest.test_ssh;8000-257a: PASS (4.26 s)
(45/81) network_test.py:NetworkTest.test_ssh;9000-74de: PASS (4.36 s)
(46/81) network_test.py:NetworkTest.test_scp;1500-d20e: PASS (19.46 s)
(47/81) network_test.py:NetworkTest.test_scp;2000-3295: PASS (19.64 s)
(48/81) network_test.py:NetworkTest.test_scp;3000-8d15: PASS (19.79 s)
(49/81) network_test.py:NetworkTest.test_scp;4000-a8bf: PASS (19.64 s)
(50/81) network_test.py:NetworkTest.test_scp;5000-fe4f: PASS (19.50 s)
(51/81) network_test.py:NetworkTest.test_scp;6000-da64: PASS (19.21 s)
(52/81) network_test.py:NetworkTest.test_scp;7000-354d: PASS (19.80 s)
(53/81) network_test.py:NetworkTest.test_scp;8000-257a: PASS (19.39 s)
(54/81) network_test.py:NetworkTest.test_scp;9000-74de: PASS (19.48 s)
(55/81) network_test.py:NetworkTest.test_jumbo_frame;1500-d20e: PASS (6.60 s)
(56/81) network_test.py:NetworkTest.test_jumbo_frame;2000-3295: PASS (7.23 s)
(57/81) network_test.py:NetworkTest.test_jumbo_frame;3000-8d15: PASS (7.30 s)
(58/81) network_test.py:NetworkTest.test_jumbo_frame;4000-a8bf: PASS (7.28 s)
(59/81) network_test.py:NetworkTest.test_jumbo_frame;5000-fe4f: PASS (7.30 s)
(60/81) network_test.py:NetworkTest.test_jumbo_frame;6000-da64: PASS (7.31 s)
(61/81) network_test.py:NetworkTest.test_jumbo_frame;7000-354d: PASS (7.28 s)
(62/81) network_test.py:NetworkTest.test_jumbo_frame;8000-257a: PASS (7.27 s)
(63/81) network_test.py:NetworkTest.test_jumbo_frame;9000-74de: PASS (7.16 s)
(64/81) network_test.py:NetworkTest.test_statistics;1500-d20e: PASS (7.50 s)
(65/81) network_test.py:NetworkTest.test_statistics;2000-3295: PASS (8.15 s)
(66/81) network_test.py:NetworkTest.test_statistics;3000-8d15: PASS (8.28 s)
(67/81) network_test.py:NetworkTest.test_statistics;4000-a8bf: PASS (8.20 s)
(68/81) network_test.py:NetworkTest.test_statistics;5000-fe4f: PASS (8.24 s)
(69/81) network_test.py:NetworkTest.test_statistics;6000-da64: PASS (8.22 s)
(70/81) network_test.py:NetworkTest.test_statistics;7000-354d: PASS (8.18 s)
(71/81) network_test.py:NetworkTest.test_statistics;8000-257a: PASS (8.29 s)
(72/81) network_test.py:NetworkTest.test_statistics;9000-74de: PASS (8.21 s)
(73/81) network_test.py:NetworkTest.test_promisc;1500-d20e: PASS (11.74 s)
(74/81) network_test.py:NetworkTest.test_promisc;2000-3295: PASS (12.38 s)
(75/81) network_test.py:NetworkTest.test_promisc;3000-8d15: PASS (12.37 s)
(76/81) network_test.py:NetworkTest.test_promisc;4000-a8bf: PASS (12.53 s)
(77/81) network_test.py:NetworkTest.test_promisc;5000-fe4f: PASS (12.47 s)
(78/81) network_test.py:NetworkTest.test_promisc;6000-da64: PASS (12.43 s)
(79/81) network_test.py:NetworkTest.test_promisc;7000-354d: PASS (12.51 s)
(80/81) network_test.py:NetworkTest.test_promisc;8000-257a: PASS (12.41 s)
(81/81) network_test.py:NetworkTest.test_promisc;9000-74de: PASS (12.61 s)
RESULTS : PASS 72 | ERROR 0 | FAIL 9 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME : 762.63 s
JOB HTML : /root/avocado-fvt-wrapper/results/job-2019-11-20T19.47-cdf5864/results.html

**Comment by narasimhan-v** [comment]
@smruti77 please finalise on the review

**Comment by manvanthar** [comment]
@smruti77 

if the additional ping check was an additional check after MTU setitng, then PR #1451 should have the fix with the naming correction along with TSO changes.

I personally think 2 ping checks are not needed in Setup.  If you also feel so,  We can go ahead with this PR.

**Comment by narasimhan-v** [comment]
@manvanthar review please

**Comment by manvanthar** [comment]
yeah fine with the changes!!

File: io/net/network_test.py [hunk 1]
```diff
@@ -65,17 +65,18 @@ def setUp(self):
...
-        if not HostInfo.ping_check(self, self.iface, self.peer, "2"):
-            self.cancel("No connection to peer")
         self.mtu = self.params.get("mtu", default=1500)
         self.peer_user = self.params.get("peer_user", default="root")
         self.peer_password = self.params.get("peer_password", '*',
                                              default=None)
         self.peerinfo = PeerInfo(self.peer, peer_user=self.peer_user,
                                  peer_password=self.peer_password)
         self.peer_interface = self.peerinfo.get_peer_interface(self.peer)
+        self.mtu = self.params.get("mtu", default=1500)
```
Comment (narasimhan-v) [inline io/net/network_test.py]: @vaishnavibhat please remove this line, and we are ready to merge.

**Pull request merged**